examples/ecs/nondeterministic_system_order.rs

```diff
@@ -0,0 +1,85 @@
+//! By default, Bevy systems run in parallel with each other.
+//! Unless the order is explicitly specified, their relative order is nondeterministic.
+//!
+//! In many cases, this doesn't matter and is in fact desirable!
+//! Consider two systems, one which writes to resource A, and the other which writes to resource B.
+//! By allowing their order to be arbitrary, we can evaluate them greedily, based on the data that is free.
+//! Because their data accesses are **compatible**, there is no **observable** difference created based on the order they are run.
+//!
+//! But if instead we have two systems mutating the same data, or one reading it and the other mutating,
+//! than the actual observed value will vary based on the nondeterministic order of evaluation.
```

Comment:
May be a typo ? I'm not a native english speaker though 😛 
```suggestion
//! then the actual observed value will vary based on the nondeterministic order of evaluation.
```